src/js/components/NewTabMainDiv.jsx

```diff
@@ -50,66 +59,93 @@ let bg = randomPick([
 
 	// iframe src change?
 	// https://stackoverflow.com/posts/17316521/revisions
 
 	return (
 		<BG src={bg.src} fullscreen opacity={0.9}>
-			<div className="container">				
-				
-				<Card id="score" body className="pull-right">
-					<LoginAccountControl />
-					£1,000,000 raised
-				</Card>
-
-				<Card body>
-					<Form onSubmit={google} inline className="flex-row" >
-						<PropControl type="search" prop="q" path={['widget', 'search']} className="flex-grow" /><Button color="secondary" onClick={google}>Search</Button>
-					</Form>
-				</Card>
-
-				<Row>
-					<h2 className='text-dark bg-light'>What charity would you like to support?</h2>
-					{charities.map(c => <NewTabCharityCard key={c} cid={c} />)}
-				</Row>
-
-				<Card body><CardTitle></CardTitle>
-					<BannerAd />
-				</Card>
+			<div className="position-fixed p-3" style={{top: 0, left: 0, width:"100vw", zIndex:10}}>
+				<div className="d-flex justify-content-end">
+					<AccountMenu/>
+				</div>
+			</div>
+			<DonationCount className="mt-2"/>
+			<div className="flex-column justify-content-end align-items-center position-absolute unset-margins" style={{top: 0, left: 0, width:"100vw", height:"100vh"}}>
+				<div className="container h-100 flex-column justify-content-center unset-margins">
+
+					<div className="w-100 pb-3">
+						<div className="tab-search-container mx-auto">
+							<Search ref={searchRef} />
+
+						</div>
+					</div>
 
-				{C.SERVER_TYPE !== 'local' ? <DevLink href="http://localmy.good-loop.com/newtab.html">Local Version</DevLink> : <Badge>local</Badge>}
-				{C.SERVER_TYPE !== 'test' ? <DevLink href="https://testmy.good-loop.com/newtab.html">Test Version</DevLink> : <Badge>test</Badge>}
-				{!C.isProduction() ? <DevLink href="https://my.good-loop.com/newtab.html">Production Version</DevLink> : <Badge>live</Badge>}
+					<Row className="justify-content-center">
+						{charities.map(c => <NewTabCharityCard key={c} cid={c} />)}
+					</Row>
 
-				<NewTabFooter />
+					<BannerAd />
+
+					{/*<div>
+						<Card body>
+							<CardTitle></CardTitle>
+							<BannerAd />
+						</Card>
+
+						{C.SERVER_TYPE !== 'local' ? <DevLink href="http://localmy.good-loop.com/newtab.html">Local Version</DevLink> : <Badge>local</Badge>}
+						{C.SERVER_TYPE !== 'test' ? <DevLink href="https://testmy.good-loop.com/newtab.html">Test Version</DevLink> : <Badge>test</Badge>}
+						{!C.isProduction() ? <DevLink href="https://my.good-loop.com/newtab.html">Production Version</DevLink> : <Badge>live</Badge>}
+					</div>*/}
+				</div>
 			</div>
+			<NewTabFooter />
+			<TutorialCard element={searchRef}/>
 		</BG>);
 };
 
-const LoginAccountControl = () => {
-	if ( ! Login.isLoggedIn()) {		
-		return <LoginLink />;
-	}
-	let user = Login.getUser();
-	return <div>
-		{user.name || user.id}
-		<small><LogoutLink /></small>
-	</div>;
+const Search = () => {
+	return (<>
```

Comment:
Hm - We should tell the user that this search is via Ecosia, and will raise money for planting trees.

Reply:
Added placeholder text